dv/uvm/core_ibex/sim.py

```diff
@@ -478,13 +478,14 @@ def gen_cov(base_dir, simulator, lsf_cmd):
     """
     # Compile a list of all output seed-###/rtl_sim/test.vdb directories
     dir_list = []
-    for entry in os.scandir(base_dir):
-        vdb_path = "%s/%s/rtl_sim/test.vdb" % (base_dir, entry.name)
-        if 'seed' in entry.name:
-            logging.info("Searching %s/%s for coverage database" %
-                         (base_dir, entry.name))
-            if os.path.exists(vdb_path):
-                dir_list.append(vdb_path)
+
+    # All generated coverage databases will be named "test.vdb"
+    vdb_dir_name = "test.vdb"
+
+    for path, dirs, files in os.walk(base_dir):
+      if vdb_dir_name in dirs:
+        dir_list.append(os.path.join(path, vdb_dir_name))
```

Comment:
This file uses 4-space indentation elsewhere: can you update this to match? Also, would it make sense to include a `logging.info` call to show the database we're adding? The code that's being replaced (sort of) had that, and it might be handy for working out what's going into the result.

Reply:
Sure, I can add that logging call back in.